tools/generate_tags_and_topics.py

```diff
@@ -1,6 +1,9 @@
 #!/usr/bin/env python
```

Comment:
these files were formatted in my editor, because they typically aren't picked up by ruff.